src/SystemView.cpp

```diff
@@ -453,8 +459,20 @@ void SystemView::PutOrbit(const Orbit *orbit, const vector3d &offset, const Colo
 			break;
 	}
 
+	Color* colors = new Color[num_vertices];
```

Comment:
Maybe use a `std::unique_ptr` or a `std::vector` here instead. Just for convention.